README.md

```diff
@@ -200,70 +201,65 @@ let credential = await navigator.credentials.get({
       {
         origin: "https://login.idp.net",
       },
+    ]
   }
 });
 ```
 
-or embeds an IdP iframe or other resources that want to access unpartitioned cross-site storage.
+We need to bypass the requirement for user interaction here if we had user interaction to force the navigation but redirected the current tab instead of opening a popup.
```

Comment:
This seems a bit involved so it might make sense to track / write up this requirement and how it would work in a separate issue vs. just mentioning it here. So I would just remove this and file a new issue instead.